Main/src/IO/TempData.cs

```diff
@@ -171,15 +173,19 @@ protected override void DisposePath(string path, bool disposing)
 		#endregion
 
 		#region Factory methods
-		private static string GetTempName() => Guid.NewGuid() + ".tmp";
+		/// <summary>Creates a temporary file or directory name</summary>
+		/// <returns>The temporary name</returns>
+		/// <remarks>The resulting name is a local name (does not include a base path)</remarks>
+		public static string GetTempName() => Guid.NewGuid() + ".tmp";
```

Comment:
It is useful to construct temporary files in custom locations. So, it is useful to make it public.